src/Sarif/SdkResources.resx

```diff
@@ -243,4 +243,7 @@
   <data name="ERR997_AllRulesExplicitlyDisabled" xml:space="preserve">
     <value>All rules were explicitly disabled so there is no work to do.</value>
   </data>
+  <data name="ErrorMatchedResultsRequiresNonNullResult" xml:space="preserve">
+    <value>MatchedResults requires at least one non-null Result.</value>
```

Comment:
>MatchedResults [](start = 11, length = 14)

I wouldn't have expected this resource/exception string to be in a resx file since I'm not sure an end user would know what to do with it. i.e. it seems like an internal exception. ... I only ask in case there's cost to adding to the resx file; localization, etc.

Reply:
You're right, this one isn't really user facing. I'm not concerned about the runtime cost of extracting a rarely-used string from resources, but there's no sense spending money for a translator to localize it.

---
In reply to: [296904795](https://github.com/microsoft/sarif-sdk/pull/1542#discussion_r296904795) [](ancestors = 296904795)